app/assets/javascripts/controllers/project_roles_ctrl.js

```diff
@@ -0,0 +1,69 @@
+samson.controller('ProjectRolesCtrl', function($scope, $element, $filter, userProjectRoleFactory, projectRoleFactory, projectRolesService, messageCenterService) {
+  $scope.project_role = {};
+  $scope.roles = [];
+
+  $scope.initModel = function() {
+    $scope.project_role = userProjectRoleFactory.buildFromDom($element[0]);
```

Comment:
I'd probably leave this named as 'build' since you're passing in DOM Element anyway. No need to make it that explicit, and seeing DOM made me think you were going to walk a whole tree of elements rather then just reading the data fields of that element.